index.js

```diff
@@ -156,6 +182,27 @@ app.on('ready', () => {
   }
 });
 
+// Put notes opened in a new window into focus mode
+ipcMain.on('note-frame-loaded', event => {
+  if (event.sender !== mainWindow.webContents) {
+    event.sender.send('focus-mode');
+  }
+});
+
+// Open a sub window, offset from calling window
+// Set width equal to the approximate note frame in main window
+ipcMain.on('open-sub-window', event => {
+  const noteWindow = BrowserWindow.fromWebContents(event.sender);
```

Comment:
We can create the new note window based on the dimension of the user's display/work-area size, thus always achieving the same window:screen ratio, regardless of the hardware. In adddtion, instead of using indices, we can use array destructuring in order to get the `x` & `y` window coordinates, and also we can provide the window with a new unique title, instead of calling it `Tusk`, using the name of the note as main identifying element:

```js
// New `ipcMain.on('open-sub-window')`
ipcMain.on('open-sub-window', event => {
  const {workAreaSize} = electron.screen.getPrimaryDisplay();
  const {width: screenWidth, height: screenHeight} = workAreaSize;

  const noteWindow = BrowserWindow.fromWebContents(event.sender);
  const [x, y] = noteWindow.getPosition();

  const noteTitle = noteWindow.webContents.getTitle().replace(' | Evernote Web', '');

  const options = {
    x: x + 50,
    y: y - 50,
    title: `${app.getName()} - ${noteTitle || || `New Note`}`,
    width: Math.round((screenWidth * 3) / 5),
    height: Math.round((screenHeight * 3) / 5)
  };

  subWindows.push(createSubWindow(noteWindow.webContents.getURL(), options));
});
```